psi4/driver/qcdb/dashparam.py

```diff
@@ -246,42 +246,43 @@ def dftd3_coeff_formatter(dashlvl, dashcoeff):
     #   d3mbj:   s6 a1 s8 a2 alpha6=None version=6
 
     dashlvleff = dash_alias['-' + dashlvl][1:] if ('-' + dashlvl) in dash_alias.keys() else dashlvl
+    dashformatter = """{:12.6f} {:12.6f} {:12.6f} {:12.6f} {:12.6f} {:6}\n"""
 
     if dashlvleff.lower() == 'd2p4':
-        returnstring = '%12.6f %12.6f %12.6f %12.6f %12.6f %6d\n' % \
-            (dashcoeff['s6'],
+        returnstring = dashformatter.format(
```

Comment:
Ok, so I agree with your assertion that the `.format` is cleaner than the old `%` syntax now.

Reply:
Ah, but mixed args & kwargs spec is so [much cleaner](https://github.com/psi4/psi4/pull/834/files#diff-c4547a95fd0ad2c0d6cb9bcf0d401046R641) with `format`. I still cling to `printf` in cpp, but we'll just have to disagree over `%` vs `format` in py. I love it. :-)